x/warp/predicate_test.go

```diff
@@ -422,15 +421,12 @@ func TestWarpSignatureWeightsDefaultQuorumNumerator(t *testing.T) {
 		int(params.WarpQuorumDenominator) + 1,
 	} {
 		var (
-			predicateBytes       = createPredicate(numSigners)
-			expectedPredicateErr error
+			predicateBytes           = createPredicate(numSigners)
+			expectedPredicateResults = set.NewBits()
 		)
 		// If the number of signers is less than the params.WarpDefaultQuorumNumerator (67)
```

Comment:
nit: could we update this comment similar to this one: https://github.com/ava-labs/subnet-evm/pull/823/files#diff-52b509c54180f89eb1daa8899e8c1404dcca86bee2bc984bba74961ff86c6762R523-R524